molecule/default/verify.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Verify
+  hosts: all
+  tasks:
+    - name: Verify that host file exists.
+      stat:
+        path: "/etc/hosts"
+      register: host_file
+
+    - name: Fail if host file does not exist
+      fail:
+      when: not host_file.stat.exists
```

Comment:
Mhm not entirely sure about the naming of these two tasks... Feels like the first task name is more suitable for the second task here...

Reply:
renamed!